src/source/w3iorsmd.F90

```diff
@@ -933,6 +903,68 @@ SUBROUTINE W3IORS ( INXOUT, NDSR, DUMFPI, INTYPE, IMOD )
 !/ End of W3IORS ----------------------------------------------------- /
 !/
       END SUBROUTINE W3IORS
+!/ ------------------------------------------------------------------- /
+#ifdef CESMCOUPLED
+      SUBROUTINE CESM_REST_FILENAME(LWRITE, FNAME) 
+
+        USE WAV_SHR_MOD , ONLY : CASENAME, INITFILE, INST_SUFFIX 
+        USE WAV_SHR_MOD , ONLY : RUNTYPE, STDOUT, ROOT_TASK
+        USE W3WDATMD    , ONLY : TIME
+        USE W3SERVMD    , ONLY : EXTCDE
+
+        ! input/output variables
+        logical, intent(in)           :: lwrite
+        character(len=*), intent(out) :: fname
+
+        ! local variables
+        integer :: yy,mm,dd,hh,mn,ss,totsec
+        logical :: exists
+        logical :: lread
+        !----------------------------------------------
+
+        ! create local lread logical for clarity
+        lread = (lwrite == .false.)
```

Comment:
GNU compiler is complaining about the usage of `==` here. Please hange it with `.eqv` operator instead.

Reply:
Thanks for checking this. I have replaced this if more clear if/then statements.

Reply:
Instead of if/then, could I suggest `lread = .not. lwrite`?

Reply:
@mnlevy1981  Agreed! I am merging this pr as is for now, but will update this later.